backend/pkg/api/enums/validator_dashboard_enums.go

```diff
@@ -270,6 +270,66 @@ var VDBManageValidatorsColumns = struct {
 	VDBManageValidatorsWithdrawalCredential,
 }
 
+// ----------------
+// Validator Dashboard Archived Reasons
+
+type VDBArchivedReason int
+
+var _ EnumFactory[VDBArchivedReason] = VDBArchivedReason(0)
```

Comment:
Does this need to be an `EnumFactory`? If we're not validating the DB value in https://github.com/gobitfly/beaconchain/pull/692/files#diff-600151bf2b435c255348f0c1a82d41fcde5c86a5a0fec8040be3b6eb5ff1de1fR100, I don't think `NewFromString()` is going to be used anywhere else

Reply:
After trying to understand more how `EnumFactory` is used I agree and changed it to the normal `Enum`.